README.md

```diff
@@ -156,6 +156,16 @@ options:
 Each worksheet has an entry `Test Active` which can be either true or false.  If set to false, the worksheet will be skipped, unless it is
 the named worksheet using the `-w` command line switch.
 
+## Integration tests (cucumber)
+
+We are beginning the process of transitioning from "ispec" to cucumber. During the transition we have tests defined in both formats. Newer tests are written as cucumber features in the `features` folder.
```

Comment:
we cannot get rid of ispec without approval from Dave's input would say as he has used them a lot to test expected values in the past

Reply:
Fair point. I will reword the readme to remove the suggestion that getting rid of ispec is a definite.